docs/getting-started.md

```diff
@@ -138,7 +142,7 @@ application repositories
   was not intialized)
   ```
   VARIABLE_GROUP_NAME=<my-vg-name>
-  spk project create-variable-group $VARIABLE_GROUP_NAME -r $ACR_NAME -u $SP_APP_ID -t $SP_TENANT -p $SP_PASS
+  spk project create-variable-group $VARIABLE_GROUP_NAME -d $HLD_REPO_URL -r $ACR_NAME -u $SP_APP_ID -t $SP_TENANT -p $SP_PASS
```

Comment:
@dennisseah is `-d $HLD_REPO_URL` required? It should read from the config for the value.

Reply:
@mtarng - you are correct. It is an optional parameter and falls back to SPK config when it is not specified.

Reply:
> @dennisseah is `-d $HLD_REPO_URL` required? It should read from the config for the value.

i got an error if I do not provide it.

Reply:
@sarath-p is able to run it without -d option. I will amend this PR.